tasks/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 
 - name: Initialize
-  deploy_helper: "path={{ project_root }} state=present"
+  deploy_helper: "path={{ project_root }} state=present shared_path={{ project_shared_path|default(project_shared_path) }}"
```

Comment:
You don't have to use the default filter in this case. The variable `project_shared_path` is always set, with the lowest possible precedence, in the role defauls.
So this is enough:

    `shared_path={{ project_shared_path }}`